ui/DesignSystem/Component/Sidebar.svelte

```diff
@@ -154,6 +155,13 @@
     </li>
   </ul>
   <ul class="bottom">
+    <li class="item indicator" data-cy="search">
+      <Tooltip value="Navigate to a project">
+        <div on:click|stopPropagation={() => dispatch('opensearch')}>
```

Comment:
This is not centred anymore. You'll need to apply the styles to this `div` that are applied to the `<a />` for it to be properly styled

Reply:
something like this: 
```
a,
  .icon-container {
    display: flex;
    align-items: center;
    justify-content: center;
    width: 100%;
    height: 100%;
  }
```

Reply:
Good catch; I made it an anchor tag so it's consistent with the others.